package.json

```diff
@@ -80,7 +80,6 @@
     "cheerio": "^1.0.0-rc.2",
     "codemirror": "^5.31.0",
     "css-tree": "^1.0.0-alpha.28",
-    "dom-to-image": "^2.6.0",
```

Comment:
Is this intended?

Reply:
In the beginning, I want to change `Katex` to SVG, But is has a low performance.
So I used HTML which generated by `katex.renderToString` directly.